lib/asset/transferasset.js

```diff
@@ -210,7 +209,7 @@ class TransferAsset {
         };
 
         if (this.metadata && this.metadata.contentType) {
-            data.mimeType = this.metadata.contentType || MIMETYPE.APPLICATION_OCTET_STREAM;
+            data.mimeType = this.metadata.contentType;
```

Comment:
This line didn't do anything, this PR takes care of mimetype handling in both the initiate as well as the complete upload code.